modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -56,12 +73,25 @@ nameserver=4.4.4.41
 ----
 
 a|Specify multiple network interfaces by specifying multiple `ip=` entries.
+
+If the additional network gateway is different from the primary network gateway, the default gateway must be the primary network gateway.
```

Comment:
@miabbott Could you please take a look and confirm that it's correct for x86. Thanks